PETWorks/arx.py

```diff
@@ -83,7 +97,7 @@ def getQiIndices(dataHandle: str) -> list[int]:
     return qiIndices
 
 
-def findAnonymousLevel(hierarchy: list[list[str]], value: str) -> int:
+def findAnonymousLevel(hierarchy: Hierarchy, value: str) -> int:
```

Comment:
IMHO, we describe the typing as `Hierarchy` is not correct. It's a Java array, so I recommend describing it as `JavaArray`, which is from the `py4j.java_collection` module.